command/debug/debug_test.go

```diff
@@ -63,6 +62,16 @@ func TestDebugCommand(t *testing.T) {
 	require.Equal(t, 0, code)
 	require.Equal(t, "", ui.ErrorWriter.String())
 
+	expected := fs.Expected(t,
+		fs.WithDir("debug",
+			fs.WithFile("agent.json", "", fs.MatchAnyFileContent),
+			fs.WithFile("host.json", "", fs.MatchAnyFileContent),
+			fs.WithFile("index.json", "", fs.MatchAnyFileContent),
+			fs.WithFile("members.json", "", fs.MatchAnyFileContent),
```

Comment:
The target name is cluster, I think we should rename both for a better user experience. I'm not sure though if renaming a debug subcommand is considered a breaking change.
Also, I believe we have a reference to it as a cluster target in the documentation

Reply:
Good question! I believe I have handled that in the second commit, specifically here: https://github.com/hashicorp/consul/pull/10804/commits/58af70c2207945ce7c465279f78cd2eec122d1fb#diff-e4c25813d9467f20b9374e5dd16394ccf77aaf1db8fee3d1c7bba52b716b3cdeR551-R552

The way it should work now is that:
* both `cluster` and `members` should be accepted values for the `-capture` flag, and they should both do the same thing
* the help text should say `members` now

So I believe this handles backwards compatibility and ensures the flag values match the filename. 

But good call on the docs! I missed an update to https://www.consul.io/commands/debug#capture-targets 

I'll add a commit to this PR to make that change.